src/python/pants/backend/helm/utils/docker.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class ImageRef:
```

Comment:
This seems like something that might already exist in the docker backend? cc @kaos

Reply:
not a parser for Docker image refs, but have to admit that I added this type in the early stages of the PR and it's quite likely I can now get rid of it and just treat them as `str`